CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## [v1.0.54](https://github.com/stargate/stargate/tree/v1.0.54) (2022-04-18)
+
+[Full Changelog](https://github.com/stargate/stargate/compare/v2.0.0-ALPHA-7...v1.0.54)
```

Comment:
This isn't right. v2 shouldn't be in the history

Reply:
Right, something went wrong, let me check...

Reply:
I just don't think the tool we are using to generate changelogs is equipped to address our situation of two active release streams, so manual cleanup is required for each v1 and v2 release unfortunately. But we won't be in this situation forever.